lib/grammers-client/src/client/messages.rs

```diff
@@ -262,6 +264,50 @@ impl SearchIter {
         self
     }
 
+    /// Restricts results to messages sent by the logged-in user
+    pub fn sent_by_self(mut self) -> Self {
```

Comment:
Would probably make sense to accept anything that can be converted into a `PackedChat` instead (like the rest of methods do).